src/lib.rs

```diff
@@ -873,7 +946,24 @@ impl JsonlToCiff {
             // Sum of tf's in this doc => doc_length
             let mut doc_length = 0i64;
             for (term, score) in jdoc.vector {
-                let tf = score.round() as i32;
+                let tf = if self.quantize {
+                    // 8-bit scalar quantization: map [min_score, max_score] to [1, 256]
```

Comment:
This also should be a separate function. And again, it will allow us to easily test different combinations and corner cases.

Interesting things to test:
* If score=min, then quantized score is 1
* If score=max, then quantized score is 255
* Some other value